src/v/transform/logging/log_manager.cc

```diff
@@ -274,11 +284,32 @@ void manager<ClockType>::enqueue_log(
         return res;
     };
 
+    auto get_probe =
+      [this](std::string_view name) -> std::optional<logger_probe*> {
+        auto res = _logger_probes.find(name);
+        if (res == _logger_probes.end()) {
+            auto [it, ins] = _logger_probes.emplace(
+              ss::sstring{name.data(), name.size()},
+              std::make_unique<logger_probe>());
+            if (!ins) {
```

Comment:
Can this really happen? It's kind of annoying to have to handle

Reply:
Yeah i suppose not. Not really what the flag is for